Sources/HummingbirdCore/Server/Server.swift

```diff
@@ -272,6 +291,17 @@ public actor Server<ChildChannel: ServerChildChannel>: Service {
 
 /// Protocol for bootstrap.
 protocol ServerBootstrapProtocol {
+    /// Initialize the `ServerSocketChannel` with `initializer`. The most common task in initializer is to add
+    /// `ChannelHandler`s to the `ChannelPipeline`.
+    ///
+    /// The `ServerSocketChannel` uses the accepted `Channel`s as inbound messages.
+    ///
+    /// - note: To set the initializer for the accepted `SocketChannel`s, look at `ServerBootstrap.childChannelInitializer`.
+    ///
+    /// - parameters:
+    ///     - initializer: A closure that initializes the provided `Channel`.
+    func serverChannelInitializer(_ initializer: @escaping @Sendable (Channel) -> EventLoopFuture<Void>) -> Self
```

Comment:
Am I missing an implementation for HTTP/1.1 and HTTP/2?

Reply:
Not sure what you mean here? This is here to abstract what bootstrap I am using eg Posix or TS